queue_consumer.yaml

```diff
@@ -7,6 +7,18 @@ parameters:
     displayName: build id
     type: string
     default: "__not_supplied"
+  - name: prod_storage_account_name
+    displayName: Storage Account Name
+    type: string
+    default: mobyreleases
+  - name: index_container
+    displayName: Storage Container for storing index
+    type: string
+    default: index
+  - name: rebuild
```

Comment:
Can you remove this one?
I don't think we should ever do a destructive rebuild from this pipeline.

Reply:
done